deploy/flashloan/1_flashAngleGovernance.ts

```diff
@@ -11,8 +12,15 @@ import {
   ProxyAdmin__factory,
   Treasury,
   Treasury__factory,
-} from '../typechain';
-import params from './networks';
+} from '../../typechain';
+import { parseAmount } from '../../utils/bignumber';
+
+const flashLoanParams = {
+  // 3m at the moment, should not be too big with respect to the total agEUR in circulation
```

Comment:
Maybe lower with stUSD as we don't know the growth yet

Reply:
Adjusted